src/components/SupportIndicator/SupportIndicator.scss

```diff
@@ -0,0 +1,31 @@
+.SupportIndicatorContainer {
```

Comment:
If I understood correctly, we are keeping this SCSS file? If yes, let's keep it, but we should remember to replace this with MUI styles in the future, if we agree to use MUI styles overall